addons/survey/models/survey_question.py

```diff
@@ -170,15 +170,11 @@ def default_get(self, fields):
             'All "Is a scored question = True" and "Question Type: Date" questions need an answer')
     ]
 
-    @api.onchange('validation_email')
-    def _onchange_validation_email(self):
-        if self.validation_email:
-            self.validation_required = False
-
-    @api.onchange('is_page')
-    def _onchange_is_page(self):
-        if self.is_page:
-            self.question_type = False
+    @api.depends('is_page')
+    def _compute_question_type(self):
+        for question in self:
+            if question.is_page:
```

Comment:
if not question.question_type or question.is_page ?

Reply:
It did not cause issues on the runbot (maybe because it's not an actual boolean?)
But better safe than sorry, I added it anyway.